### DISCUSSION TITLE
Combobox: No results found while searching

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1730] [General] [answered]
Do we provide any feedback to the user if no results have been found or match to what they typed?

In the example below the dropdown goes empty. I'm wondering if we can provide some feedback
![Screen Shot 2021-08-03 at 10 36 36 AM](https://user-images.githubusercontent.com/81593493/128053393-ae7871c5-23b0-4556-a1c2-fdeddffdf28b.png)

**Comment by richbachman** [comment, marked as answer]
Hi @wilfredomerced 

We leave that pretty much open to whatever teams want to do with an empty or no results state. The only requirement is that it needs to be a clickable item. You can see an example in the Console search:

- Empty state: https://share.getcloudapp.com/Bluxg15l
- No results state: https://share.getcloudapp.com/eDuRzPPK

Let us know if you have any other questions.

**Comment by mattcullenmeyer** [comment]
I handled a no results state the following way. Curious to hear if something like this is valid and/or if there are other suggestions.

```
const items = data.length ===  0 ? ['No results found'] : data;

<Combobox  
  items={items} 
  ...  
/>
```

**Reply by loreina** [reply]
Hi @mattcullenmeyer, your implementation here looks valid. 👍
